src/features/game/lib/goblinMachine.ts

```diff
@@ -126,10 +147,7 @@ export function startGoblinVillage(authContext: AuthContext) {
                   ),
               }),
             },
-            onError: {
```

Comment:
Without this, an error message would not be handled. The state machine would not know how to handle the error. Is there a reason this was removed?